common/tools/mcp-server/src/tools/prettier.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { CallToolResult } from "@modelcontextprotocol/sdk/types.js";
+import { z } from "zod";
+import prettier from "prettier";
+import { readFile } from "node:fs/promises";
+
+export async function checkCode(code: string): Promise<boolean> {
+  return prettier.check(code, {
```

Comment:
can we parse our prettier config here? https://github.com/Azure/azure-sdk-for-js/blob/main/.prettierrc.json